[rpc] fix test_backward_node_failure flakiness

rohan-varma opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#31588 [rpc] fix test_backward_node_failure flakiness**

Per title. This test can sometimes fail with a different error regex
than the one that is currently tested, so add this error regex to make the test
pass consistently.

Differential Revision: [D19222275](https://our.internmc.facebook.com/intern/diff/D19222275/)

rohan-varma committed:
[rpc] fix test_backward_node_failure flakiness

Per title. This test can sometimes fail with a different error regex
than the one that is currently tested, so add this error regex to make the test
pass consistently.

Differential Revision: [D19222275](https://our.internmc.facebook.com/intern/diff/D19222275/)

[ghstack-poisoned]

rohan-varma committed:
Update on "[rpc] fix test_backward_node_failure flakiness"


Per title. This test can sometimes fail with a different error regex
than the one that is currently tested, so add this error regex to make the test
pass consistently.

Differential Revision: [D19222275](https://our.internmc.facebook.com/intern/diff/D19222275/)

[ghstack-poisoned]

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit 85279549:


* **1/1** broken upstream at merge base 218cfd56 (see [grid view](https://dr.pytorch.org/master-timeline.html?count=1&sha1=218cfd568da4dffaa0129acfc3c3f33c5f0bd0d4&should_suppress_scheduled_builds=true&should_suppress_fully_successful_columns=true&max_columns_suppress_successful=35))
  <details>
  <summary>
  You may want to rebase on the <code>viable/strict</code> branch (expand for instructions)
  </summary>
  
  If your commit is newer than `viable/strict`, you can try basing on an older, stable commit:
  ```
  git fetch origin viable/strict
  git rebase --onto viable/strict $(git merge-base origin/master HEAD)
  ```
  If your commit is older than `viable/strict`:
  ```
  git fetch origin viable/strict
  git rebase viable/strict
  ```
  Check out the [recency history](https://dr.pytorch.org/master-viable-commits.html) of this "viable master" tracking branch.
  
  </details>
  
* **0/1** failures introduced in this PR


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=852795492975e3aa34850c10d3bb923688aff6bd).

### :construction: 1 upstream failure recognized by patterns:
These builds matched patterns, but were probably caused by upstream breakages:
* [pytorch_linux_xenial_py3_clang5_asan_test](https://dr.pytorch.org/build-details.html?build_id=78594435)



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

rohan-varma committed:
Update on "[rpc] fix test_backward_node_failure flakiness"


Per title. This test can sometimes fail with a different error regex
than the one that is currently tested, so add this error regex to make the test
pass consistently.

Differential Revision: [D19222275](https://our.internmc.facebook.com/intern/diff/D19222275/)

[ghstack-poisoned]

rohan-varma commented:
Test failures are unrelated.
